Mathlib/RingTheory/Polynomial/Cyclotomic/Eval.lean

```diff
@@ -119,7 +119,7 @@ theorem cyclotomic_pos_and_nonneg (n : ℕ) {R} [LinearOrderedCommRing R] (x : R
     simp [cyclotomic_zero, cyclotomic_one, cyclotomic_two, succ_eq_add_one, eval_X, eval_one,
       eval_add, eval_sub, sub_nonneg, sub_pos, zero_lt_one, zero_le_one, imp_true_iff, imp_self,
       and_self_iff]
-  · constructor <;> intro <;> set_option tactic.skipAssignedInstances false in norm_num; linarith
+  · constructor <;> intro <;> set_option tactic.skipAssignedInstances false in linarith
```

Comment:
Check if you can remove the option as well

Reply:
Seems to work locally, thanks for the suggestion!

The next step obvious step is an "unused `set_option`" linter.